Main/display_notificiation.ino

```diff
@@ -0,0 +1,14 @@
+void getNotification() {

```

Comment:
what about keeping the name of ino files same as function?

Reply:
Oh you meant something like `getNotification.ino`?

The reason behind naming files `display_*` is I thought keeping `display` as a prefix when it comes to display related file could be useful when navigating in Arduino IDE cause it just lists files in alphabetical order. 

Maybe subfolder with straight naming scheme could be better but I'm not sure arduino IDE's recursive subfolder compilation.. It doesn't even show the tree structure